uwblueprint/child-witness-centre
# Sohail/remove google auth

## Notion Ticket Link

[Remove Google Auth for Login](https://www.notion.so/uwblueprintexecs/Remove-Google-Auth-for-Login-f7520a768f8b402e830e9ab8cce0a27a?pvs=4)



## Implementation Description
* Removed ability to login with Google Auth. Removed google auth related functions/types


## Steps to Test
1. Test signing up, ensure you receive a verification email (use 'youremail+[anythingyouwant]@uwblueprint.org' eg. 'sohailsayed+1@uwblueprint.org' to signup multiple times w the same email)
2. Test logging in once signed up and verified


## Screenshots (optional)
<img width="1432" alt="image" src="https://github.com/uwblueprint/child-witness-centre/assets/44615627/6e31f703-8cb9-4341-8e30-7091ef02bbd0">
Google sign-in option should be gone.

## Checklist
- [x ] My PR name is descriptive and in imperative tense

## Linting & Formatting
### Mac
```bash
docker exec -it CWC-backend /bin/bash -c "yarn fix"
docker exec -it CWC-frontend /bin/bash -c "yarn fix"
```

### Windows
```bash
docker exec -it CWC-backend bash -c "yarn fix"
docker exec -it CWC-frontend bash -c "yarn fix"
```
